src/components/TxBuilderAttributes.js

```diff
@@ -9,11 +9,15 @@ import TimeBoundsPicker from './FormComponents/TimeBoundsPicker';
 import {connect} from 'react-redux';
 import {StrKey} from 'stellar-sdk';
 import NETWORK from '../constants/network';
-import {fetchSequence} from '../actions/transactionBuilder';
+import {fetchSequence, fetchBaseFee} from '../actions/transactionBuilder';
 
-export default function TxBuilderAttributes(props) {
+function TxBuilderAttributes(props) {
   let {onUpdate, attributes} = props;
 
+  useEffect(() => {
+    props.dispatch(fetchBaseFee(props.horizonURL))
```

Comment:
Nitpick: For consistency, it would be nice to add `horizonURL` to the destructuring above.